--- a/src/aic8800_fdrv/rwnx_platform.c
+++ b/src/aic8800_fdrv/rwnx_platform.c
@@ -1279,7 +1279,8 @@ static int rwnx_plat_patch_load(struct rwnx_hw *rwnx_hw)
     RWNX_DBG(RWNX_FN_ENTRY_STR);

     if(rwnx_hw->usbdev->chipid == PRODUCT_ID_AIC8800DC ||
-        rwnx_hw->usbdev->chipid == PRODUCT_ID_AIC8800DW){
+        rwnx_hw->usbdev->chipid == PRODUCT_ID_AIC8800DW ||
+        rwnx_hw->usbdev->chipid == PRODUCT_ID_AIC8801){
 #ifndef ANDROID_PLATFORM
         sprintf(aic_fw_path, "%s/%s", aic_fw_path, "aic8800DC");
 #endif
@@ -2214,7 +2215,8 @@ static int rwnx_plat_userconfig_load(struct rwnx_hw *rwnx_hw) {
  *@filename name of config
 */
 static int rwnx_plat_userconfig_load(struct rwnx_hw *rwnx_hw) {
-	if(rwnx_hw->usbdev->chipid == PRODUCT_ID_AIC8800DC){
+	if(rwnx_hw->usbdev->chipid == PRODUCT_ID_AIC8800DC ||
+        rwnx_hw->usbdev->chipid == PRODUCT_ID_AIC8801){
 		rwnx_plat_userconfig_load_8800dc(rwnx_hw);
 	}else if(rwnx_hw->usbdev->chipid == PRODUCT_ID_AIC8800DW){
         rwnx_plat_userconfig_load_8800dw(rwnx_hw);
